swagger_parser/lib/src/parser/model/normalized_identifier.dart

```diff
@@ -27,7 +27,11 @@ class NormalizedIdentifier {
       return NormalizedIdentifier._([], isPrivate: isPrivate);
     }
 
-    final words = _parseWords(workingText);
+    // Remove apostrophes before parsing so they don't split words
+    // e.g., "What's New" becomes "Whats New" -> ["Whats", "New"]
+    final textWithoutApostrophes = workingText.replaceAll("'", '');
```

Comment:
If we are targeting words that will be then a part of class name or variable, like `WhatsNewClient` it would be nice to have some common sanitization fn.
Another question on how we should treat special chars - just replace them with emptiness or `What$sNewClient`